app/js/daemonManager.js

```diff
@@ -61,6 +61,9 @@ function DaemonManager() {
 		});
 	}
 
+	/**
+	 * Checks if there is an update available
```

Comment:
It's not a jsdoc standard. Why would it help grepping to have the function name right above the function itself? Wouldn't a grep just register two lines instead of one?